users/views.py

```diff
@@ -44,3 +44,24 @@ def post(self, request):
 
         except KeyError:
             return JsonResponse({'message':'KEY_ERROR'}, status=400)
+
+class MyPageView(View):
```

Comment:
마이페이지에 해당하는 api가 이 view 하나일까요?
suggestion과 같이 UserProfileView와 같이 작성하신다면 해당 유저의 프로필을 불러오는 api 뿐만 아니라, 해당 유저의 프로필을 업데이트하는 api도 해당 view class안에서 다른 함수로만 작성해서 활용할 수 있습니다!
```suggestion
class UserProfileView(View):
```